# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/audioipc/server/Cargo.toml
# Commit: 0a7873a6b522
# Full Hash: 0a7873a6b522ce4aef9a32ae4a47f093ba33e0f4
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2020-01-26 09:48:16
# Regressor Bug: 1590249
# File Overlap Count: 3
# Description:
#   Bug 1590249 - Update audioipc to 86d49ddf.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60615
# ==============================================================================

diff -r 36f6505b50c9 -r 0a7873a6b522 media/audioipc/server/Cargo.toml
--- a/media/audioipc/server/Cargo.toml	Sat Jan 25 21:30:14 2020 +0000
+++ b/media/audioipc/server/Cargo.toml	Sat Jan 25 21:31:32 2020 +0000
@@ -9,7 +9,7 @@
 edition = "2018"
 
 [dependencies]
-audio_thread_priority = "0.20.2"
+audio_thread_priority = "0.21"
 audioipc = { path = "../audioipc" }
 cubeb-core = "0.6.0"
 futures = "0.1.18"